start.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Start script for insolvency-delta-consumer
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/company-profile-api/env
+
+    PORT="${OFFICER_DELTA_PROCESSOR_PORT}"
```

Comment:
Just realised that this env var is specifically for the officers processor - we should rename

Reply:
May need to go back and ammend other start.sh files

Reply:
changed to {COMPANY_PROFILE_API_PORT} , presume that will be set as env var.